packages/dashboard/src/portal/components/NodesTable.vue

```diff
@@ -184,6 +184,10 @@ export default class NodesTable extends Vue {
   byteToGB(capacity: number) {
     return byteToGB(capacity);
   }
+
+  byteToTB(capacity: number) {
```

Comment:
please use this method for conversion to always has the same units for the both tables https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/dashboard/src/explorer/filters/toTeraOrGigaOrPeta.ts